services/slam/server_test.go

```diff
@@ -387,6 +435,12 @@ func TestFailingServer(t *testing.T) {
 		test.That(t, getPositionResp, test.ShouldBeNil)
 		test.That(t, err, test.ShouldBeError, improperImplErr)
 
+		// Get position
+		getPositionNewReq := &pb.GetPositionNewRequest{Name: testSlamServiceName}
+		getPositionNewResp, err := slamServer.GetPositionNew(context.Background(), getPositionNewReq)
+		test.That(t, getPositionNewResp, test.ShouldBeNil)
```

Comment:
[nit] common format is to test err first but super minor point (and if that isnt the full standard elsewhere feel free to ignore)